ProductTree/ViewModels/ProductTreeRows/ActualFiniteStateRowViewModel.cs

```diff
@@ -42,34 +32,38 @@ public class ActualFiniteStateRowViewModel : CDP4CommonView.ActualFiniteStateRow
         private bool isDefault;
 
         /// <summary>
-        /// Backing field for the <see cref="OwnerShortName"/> property.
+        /// Backing field for <see cref="ModelCode"/>
         /// </summary>
-        private string ownerShortName;
+        private string modelCode;
 
         /// <summary>
-        /// Backing field for <see cref="ModelCode"/>
+        /// Backing field for <see cref="ActualState"/>
         /// </summary>
-        private string modelCode;
+        private ActualFiniteState actualState;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ActualFiniteStateRowViewModel"/> class
         /// </summary>
         /// <param name="actualFiniteState">The <see cref="ActualFiniteState"/> represented</param>
         /// <param name="session">The <see cref="ISession"/></param>
         /// <param name="containerViewModel">The container <see cref="IViewModelBase{T}"/></param>
-        public ActualFiniteStateRowViewModel(ActualFiniteState actualFiniteState, ISession session, IViewModelBase<Thing> containerViewModel)
-            : base(actualFiniteState, session, containerViewModel)
+        public ActualFiniteStateRowViewModel(ParameterBase parameterBase, ActualFiniteState actualState, ISession session, IViewModelBase<Thing> containerViewModel)
+            : base(parameterBase, session, containerViewModel)
         {
             this.IsPublishable = false;
-            this.IsDefault = this.Thing.IsDefault;
-
-            var parameterOrOverrideBaseRowViewModel = containerViewModel as ParameterOrOverrideBaseRowViewModel;
-            if (parameterOrOverrideBaseRowViewModel != null)
-            {
-                this.OwnerShortName = parameterOrOverrideBaseRowViewModel.OwnerShortName;
-            }
+            this.actualState = actualState;
```

Comment:
Set property here `this.ActualState`, not the field `actualState`. You had this right before...